libs/header/p2p/gossip_score.go

```diff
@@ -0,0 +1,32 @@
+package p2p
+
+import (
+	"time"
+
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+)
+
+// GossibSubScore provides a set of recommended parameters for header GossipSub topic, a.k.a
+// HeaderSub.
+var GossibSubScore = &pubsub.TopicScoreParams{
```

Comment:
why define these here instead of in the nodebuilder?

Reply:
So that rollkit can reuse them. It's improbable that they would need a different setup

Reply:
they can copy our code then - if our libs are really meant to be used generally then we shouldn't include our network-specific configuration options in there.

Reply:
The thing is, it's not that network specific. I don't see a problem with having a general recommendation here. Users may opt-in or opt-out. But this anyway would simplify their lives